app/webpacker/css/admin_v3/pages/edit_variant.scss

```diff
@@ -0,0 +1,55 @@
+@import "unit_popout";
+
+#edit_variant {
+
+  .popout {
+    @include unit_popout;
+
+    &__button {
+      // override popout button styles
```

Comment:
If we're refactoring, we shouldn't need to override or re-apply styles. 
Instead, can these be options to the unit_popout mixin? Or can the code that's related to the bulk products screen could be left on the bulk products css file?

Reply:
From memory, it should be the other way around ie: the bulk products screen should be the one overriding the style because it's the one using a different colour. I was a bit pressed for time to go back and fix this. I'll raise a tech debt ticket when I have a minute, that can probably be a good first issue.